ui/jobs/jobs.ts

```diff
@@ -1002,31 +1089,32 @@ class Bars {
           }
         }
       }
-    }
+    });
   }
 
-  _test() {
+  _test(): void {
     const logs = [];
     const t = '[10:10:10.000] ';
-    logs.push(t + '1A:10000000:' + this.me + ' gains the effect of Medicated from ' + this.me + ' for 30.2 Seconds.');
-    logs.push(t + '15:10000000:Tako Yaki:1D60:Embolden:10000000:' + this.me + ':500020F:4D70000:0:0:0:0:0:0:0:0:0:0:0:0:0:0:42194:42194:10000:10000:0:1000:-655.3301:-838.5481:29.80905:0.523459:42194:42194:10000:10000:0:1000:-655.3301:-838.5481:29.80905:0.523459:00001DE7');
-    logs.push(t + '1A:10000000:' + this.me + ' gains the effect of Battle Litany from  for 25 Seconds.');
-    logs.push(t + '1A:10000000:' + this.me + ' gains the effect of The Balance from  for 12 Seconds.');
+    const me = this.me ?? '';
+    logs.push(`${t}1A:10000000:${me} gains the effect of Medicated from ${me} for 30.2 Seconds.`);
+    logs.push(`${t}15:10000000:Tako Yaki:1D60:Embolden:10000000:${me}:500020F:4D70000:0:0:0:0:0:0:0:0:0:0:0:0:0:0:42194:42194:10000:10000:0:1000:-655.3301:-838.5481:29.80905:0.523459:42194:42194:10000:10000:0:1000:-655.3301:-838.5481:29.80905:0.523459:00001DE7'`);
```

Comment:
Ironically, none of these lines work anymore because I never updated them to be network lines.